bundles/org.openhab.binding.mqtt.homeassistant/src/main/java/org/openhab/binding/mqtt/homeassistant/internal/discovery/HomeAssistantDiscovery.java

```diff
@@ -31,11 +31,11 @@
 import org.eclipse.smarthome.io.transport.mqtt.MqttBrokerConnection;
 import org.openhab.binding.mqtt.discovery.AbstractMQTTDiscovery;
 import org.openhab.binding.mqtt.discovery.MQTTTopicDiscoveryService;
-import org.openhab.binding.mqtt.generic.internal.MqttBindingConstants;
-import org.openhab.binding.mqtt.internal.homeassistant.BaseChannelConfiguration;
-import org.openhab.binding.mqtt.internal.homeassistant.ChannelConfigurationTypeAdapterFactory;
-import org.openhab.binding.mqtt.internal.homeassistant.HaID;
-import org.openhab.binding.mqtt.internal.homeassistant.HandlerConfiguration;
+import org.openhab.binding.mqtt.homeassistant.generic.internal.MqttBindingConstants;
```

Comment:
Hm. I failed here. I cannot rely on an "internal" package in another bundle. That is usually not exposed

Reply:
Sorry, I don't understand what you are trying to say here 😢 What do you mean? I will address the other suggestions tomorrow!

Reply:
Nevermind. I thought because of "generic.internal.MqttBindingConstants" that I'm using a class from the generic mqtt bundle within the "internal" package which would not be allowed. I haven't seen the  "homeassistant" part in front.

Reply:
Ok!